src/transform/transform.ts

```diff
@@ -697,6 +697,11 @@ function createResponseEnvelope(codeModel: CodeModel, group: OperationGroup, op:
     }
     // create poller type info
     if (isLROOperation(op)) {
+      if (op.language.go!.paging && op.language.go!.paging.member) {
+        // implementing support for this is very complicated, and since
+        // no services at present use this pattern avoid it for now
+        throw console.error(`${op.language.go!.name}: unsupported pager-poller that uses next page operation`);
```

Comment:
So just making sure I understand, we wont support pager pollers that have the next operation? There's no current use case for this?

Reply:
Correct.  I searched through the specs repo, the only specs I found that even use a next operation are for graphrbac.  Fortunately none of them are LROs.

Reply:
And to elaborate on the problem.

Suppose you have two operations that return a `FooPagerPollerResponse`.  If each operation has its own `NextFoo()` operation with a discrete set of parameters, how do we carry/populate such parameters inside the pager-poller?

Reply:
Ok, yes that does make it more difficult. Sounds good then, let's deal with it when it comes up.